docs/AboutUs.md

```diff
@@ -22,7 +22,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 <img src="images/eeliana.png" width="200px">
 
-[[github](http://github.com/eeliana)]
+[[github](http://github.com/eeliana)] [[Portfolio](/team/eeliana.html)]
```

Comment:
```suggestion
[[github](http://github.com/eeliana)] [[Portfolio](team/eeliana.html)]
```
`/team` didnt work for mine

Reply:
Do you think using the .md is better?

Reply:
ohh probably. I saw another group using .md. can help me change also?